languages.toml

```diff
@@ -430,3 +430,12 @@ file-types = ["md"]
 roots = []
 
 indent = { tab-width = 2, unit = "  " }
+
```

Comment:
```suggestion

[[language]]
```

Reply:
Doh! I thought I did that. Must have missed it in the rebase.